src/components/swap/SwapBuyFiatButton.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('SwapBuyFiatButton.tsx', () => {
     mockuseAccountDrawer.mockImplementation(() => [false, toggleWalletDrawer])
     mockUseOpenModal.mockImplementation(() => useOpenModal)
     render(<SwapBuyFiatButton />)
-    await userEvent.click(screen.getByTestId('buy-fiat-button'))
+    await act(() => userEvent.click(screen.getByTestId('buy-fiat-button')))
```

Comment:
the way I'm reading the docs, useEvent should already handle everything `act` does: 
- https://legacy.reactjs.org/docs/test-utils.html#act
- https://testing-library.com/docs/user-event/intro/